app/controllers/api/database.php

```diff
@@ -1206,6 +1230,7 @@
             throw new Exception($validator->getDescription(), 400);
         }
 
+        $afterDocument = null;
```

Comment:
Why this addition?

If `$afterDocument` is never assigned, `$afterDocument ?? null` will result in `null` anyway 👍🏻

Reply:
Ah, you're right - I removed it in e1c7e49